hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```diff
@@ -520,9 +531,10 @@ public void createMarkers(final Path path,
   private void checkParentChainForFile(Path path, TracingContext tracingContext,
       List<Path> keysToCreateAsFolder) throws AzureBlobFileSystemException {
     AbfsHttpOperation opResult = null;
+    TracingContext tracingContext1 = new TracingContext(tracingContext);
```

Comment:
This we already do in AbfsRestOperation before calling completeExecute method.
Was that not enough?

Reply:
This was missing in databricks code, hence adding here as well. Will revert here